helix-core/src/test/java/org/apache/helix/util/TestStatusUpdateUtil.java

```diff
@@ -0,0 +1,76 @@
+package org.apache.helix.util;
+
```

Comment:
Missing apache license. Maybe you could setup apache license in your intellij?

Reply:
added.